query-connector/src/app/database-service.ts

```diff
@@ -163,3 +170,43 @@ export async function getVSACValueSet(
     } as OperationOutcome;
   }
 }
+
+/**
+ * Translates a VSAC FHIR bundle to our internal ValueSet struct
+ * @param fhirValueset - The FHIR ValueSet response from VSAC
+ * @param ersdConceptType - The associated clinical concept type from ERSD
+ * @returns An object of type InternalValueSet
+ */
+export function translateVSACToInternalValueSet(
+  fhirValueset: ValueSet,
+  ersdConceptType: ErsdConceptType,
+) {
+  const id = fhirValueset.id;
+  // does this need any interpolation?
```

Comment:
Would love answers on this and other questions here! Will remove these once we get them answered

Reply:
hm, could be worth adding, but I don't think we need to have v2/v3. I think we are using v3, which is backwards compatible with v2. Ideally, any future versions would maintain that, but out of full caution, could be worth not making that assumption.